src/layouts/Header.tsx

```diff
@@ -69,6 +70,19 @@ export default function Header() {
                 ? t("navigation.display_btc")
                 : t("navigation.display_sats")}
             </DropdownItem>
+
+            {/* Adding the documentation redirecting */}
+            <DropdownItem
+              key="documentation"
+              href="https://docs.raspiblitz.org/docs/intro"
+              target="_blank"
+              rel="noopener noreferrer"
+              color="warning"
+              startContent={<BookOpenIcon className="h-5 w-5" />}
+            >
+              Documentation
```

Comment:
Can you use a translation string here? See the other Items here for a reference. Not need to add all translations, just English is enough.